components/director/internal/model/auth_test.go

```diff
@@ -1,3 +1,321 @@
-package model
+package model_test
 
-//TODO: Test AuthInput toAuth
+import (
+	"fmt"
+	"testing"
+
+	"github.com/kyma-incubator/compass/components/director/internal/model"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestAuthInput_ToAuth(t *testing.T) {
+	// given
+	testCases := []struct {
+		Name     string
+		Input    *model.AuthInput
+		Expected *model.Auth
+	}{
+		{
+			Name: "All properties given",
+			Input: &model.AuthInput{
+				Credential: &model.CredentialDataInput{
+					Basic: &model.BasicCredentialDataInput{
```

Comment:
here you test only basic creds, you should also have a test for oauth, to ensure that `ToAuth` is converting it if needed.